ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-release-6.0.yaml

```diff
@@ -0,0 +1,207 @@
+base_images:
+  base:
+    name: release
+    namespace: openshift
+    tag: rhel-9-release-golang-1.21-openshift-4.16
+  elasticsearch-operator-src:
+    name: "5.8"
+    namespace: logging
+    tag: elasticsearch-operator-src
+  log-file-metric-exporter:
+    name: "6.0"
+    namespace: logging
+    tag: log-file-metric-exporter
+  logging-eventrouter:
+    name: "6.0"
+    namespace: logging
+    tag: logging-eventrouter
+  logging-vector:
+    name: "6.0"
+    namespace: logging
+    tag: vector
+  ubi9-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-9-release-golang-1.21-openshift-4.16
+images:
+- from: ubi9-minimal
+  inputs:
+    base:
+      as:
+      - registry.redhat.io/ubi9/go-toolset:latest
+  to: cluster-logging-operator
+- dockerfile_path: olm_deploy/operatorregistry/Dockerfile
+  from: base
+  to: cluster-logging-operator-registry
+- dockerfile_literal: |
+    FROM registry.ci.openshift.org/logging/5.8:elasticsearch-operator-src AS es-src
+    FROM registry.redhat.io/ubi9/go-toolset:latest
+    ADD . /go/src/github.com/openshift/cluster-logging-operator
+    COPY --from=es-src /go/src/github.com/openshift/elasticsearch-operator /go/src/github.com/openshift/elasticsearch-operator
+    WORKDIR /go/src/github.com/openshift/cluster-logging-operator
+    USER 0
+    RUN mkdir -p /go/src/github.com/openshift/cluster-logging-operator/bin/
+    RUN go mod download
+    RUN make tools
+    RUN chmod -R 777 /go
+    RUN make bin/functional-benchmarker
+  inputs:
+    base:
+      as:
+      - registry.redhat.io/ubi9/go-toolset:latest
+    elasticsearch-operator-src:
+      as:
+      - registry.ci.openshift.org/logging/5.8:elasticsearch-operator-src
+  to: cluster-logging-operator-e2e
+operator:
+  bundles:
+  - as: cluster-logging-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-logging/cluster-logging-operator:latest
+    with: pipeline:cluster-logging-operator
+  - pullspec: quay.io/openshift-logging/vector:latest
+    with: pipeline:logging-vector
+  - pullspec: quay.io/openshift-logging/log-file-metric-exporter:latest
+    with: pipeline:log-file-metric-exporter
+promotion:
+  to:
+  - name: "6.0"
+    namespace: logging
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
```

Comment:
Looking at [the docs](https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-ephemeral-openshift-release), I'm not sure we need this. Wouldn't it be enough to just test with "4.16 stable" as described below?